lib/libs/email/content/withdrawPackage/emailTemplates/WaiverState.tsx

```diff
@@ -0,0 +1,38 @@
+import { CommonEmailVariables, Events } from "shared-types";
+import {
+  ContactStateLead,
+  BasicFooter,
+  PackageDetails,
+  MailboxNotice,
+  Divider,
+} from "../../email-components";
+import { BaseEmailTemplate } from "../../email-templates";
+
+export const WaiverStateEmail = (props: {
+  variables: Events["WithdrawPackage"] & CommonEmailVariables;
+}) => {
+  const variables = props.variables;
+  const previewText = `Withdrawal of ${variables.authority} ${variables.id}`;
+  const heading = `This is confirmation that you have requested to withdraw the package below. The package will no longer be considered for CMS review:`;
+  return (
+    <BaseEmailTemplate
+      previewText={previewText}
+      heading={heading}
```

Comment:
I think it'd look cleaner if you simply declare the strings in the prop arguments:
```tsx
previewText={`Withdrawal of ${variables.authority} ${variables.id}`}
heading={`This is confirmation that you have requested to withdraw the package below. The package will no longer be considered for CMS review:`}
```

What do you think?

Reply:
Also, `variables` can be destructured 🤷🏻‍♂️ – that's just my preference for readability